lib/discovery.js

```diff
@@ -242,53 +220,24 @@ function mixinDiscovery(MsSQL) {
   };
 
   /**
-   * Discover primary keys for a given table
-   * @param {String} table The table name
-   * @param {Object} options The options for discovery
-   * @param {Function} [cb] The callback function
-   */
-  // MsSQL.prototype.discoverPrimaryKeys = function(table, options, cb) {
-  //   var args = this.getArgs(table, options, cb);
-  //   var owner = args.owner;
-  //   table = args.table;
-  //   options = args.options;
-  //   cb = args.cb;
-  //
-  //   var sql = this.queryPrimaryKeys(owner, table);
-  //   this.execute(sql, cb);
-  // };
-
-  /*!
    * Build the sql statement for querying foreign keys of a given table
    * @param owner
    * @param table
    * @returns {string}
-   */
-  /*
-   SELECT
-   tc.CONSTRAINT_NAME, tc.TABLE_NAME, kcu.COLUMN_NAME,
-   ccu.TABLE_NAME AS foreign_table_name,
-   ccu.COLUMN_NAME AS foreign_column_name
-   FROM
-   INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS tc
-   JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS kcu
-   ON tc.CONSTRAINT_NAME = kcu.CONSTRAINT_NAME
-   JOIN INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE AS ccu
-   ON ccu.CONSTRAINT_NAME = tc.CONSTRAINT_NAME
-   WHERE CONSTRAINT_TYPE = 'FOREIGN KEY' AND tc.TABLE_NAME='mytable';
-
    */
   MsSQL.prototype.buildQueryForeignKeys = function(owner, table) {
     var sql =
       'SELECT tc.TABLE_SCHEMA AS "fkOwner", tc.CONSTRAINT_NAME AS "fkName", tc.TABLE_NAME AS "fkTableName",' +
       ' kcu.COLUMN_NAME AS "fkColumnName", kcu.ORDINAL_POSITION AS "keySeq",' +
-      ' ccu.TABLE_SCHEMA AS "pkOwner", \'PK\' AS "pkName", ' +
-      ' ccu.TABLE_NAME AS "pkTableName", ccu.COLUMN_NAME AS "pkColumnName"' +
+      ' kcu2.TABLE_SCHEMA AS "pkOwner", rc.UNIQUE_CONSTRAINT_NAME AS "pkName",' +
```

Comment:
I don't see the `kcu2` alias defined in the SQL, such as `as kcu2`.

Reply:
That alias comes from the join on line 239.